doc/template.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>__SECTION__ Node.js __VERSION__ Manual &amp; Documentation</title>
+  <title>__SECTION__ | Node.js __VERSION__ Manual &amp; Documentation</title>
```

Comment:
+1

On Monday, August 1, 2016, silverwind <notifications@github.com> wrote:

> In doc/template.html
> <https://github.com/nodejs/node/pull/7939#discussion_r73095875>:
>
> > @@ -2,7 +2,7 @@
> >  <html lang="en">
> >  <head>
> >    <meta charset="utf-8">
> > -  <title>__SECTION__ Node.js __VERSION__ Manual &amp; Documentation</title>
> > +  <title>__SECTION__ | Node.js __VERSION__ Manual &amp; Documentation</title>
>
> Maybe drop the Manual &amp; while we're here? Long titles like this
> usually get truncated on tabs.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node/pull/7939/files/f79b4872d33cb854f5fd5661f2762c0782b98415#r73095875>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAa2eV0M2XcFXDfNPfuDQiYrWkEvxB5Wks5qbtTHgaJpZM4JZ0aC>
> .
>